files/tasks.py

```
@@ -1216,6 +1216,93 @@ def remove_media_file(media_file=None):
+                # Only process directories
+                if not os.path.isdir(dir_path):
+                    continue
+
+                # Check if directory is old enough to be considered orphaned
+                try:
+                    dir_mtime = os.path.getmtime(dir_path)
+                    if (current_time - dir_mtime) > cleanup_age_seconds:
+                        logger.info(f"Removing orphaned chunks directory: {uuid_dir}")
+                        shutil.rmtree(dir_path)
+                        chunks_cleaned += 1
+                except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)